pkg/group/repository.go

```diff
@@ -25,12 +25,11 @@ func (r repository) find(name string) (*model.Group, error) {
 		First(&group).Error
 	if err != nil {
 		if errors.Is(err, gorm.ErrRecordNotFound) {
```

Comment:
You can now join the two if statements into one. If `errors.Is()` handles `nil` gracefully, you can even only use the condition `errors.Is(err, gorm.ErrRecordNotFound)`